.projenrc.js

```diff
@@ -44,11 +53,32 @@ project.eslint.addRules({
   "import/no-extraneous-dependencies": [
     "error",
     {
-      devDependencies: ["**/setupTests.ts", "**/*.test.tsx", "**/*.test.ts"],
+      devDependencies: [
+        "**/setupTests.ts",
+        "**/*.test.tsx",
+        "**/*.test.ts",
+        "**/*.stories.*",
+      ],
       optionalDependencies: false,
       peerDependencies: true,
     },
   ],
 });
 
+// Add tasks and config for storybook
```

Comment:
Could you package all the storybook related stuff under a function instead of spread across the file?

(Would be useful to extract to a projen component later)